tox.ini

```
@@ -0,0 +1,15 @@
+[tox]
+minversion = 3.7.0
+skip_missing_interpreters = true
+envlist =
+    py{37,38,39}
+
+[testenv]
+passenv = DD_SOURCE_*  DD_DESTINATION_* PYTEST_*
+setenv =
+    DD_HTTP_CLIENT_RETRY_TIMEOUT = 300
+    MAX_WORKERS = 1
+    RECORD = {env:RECORD:false}
+usedevelop = true
+extras = tests
+commands = pytest -ra -v
```

```suggestion
commands = pytest -ra -v {posargs}
```
to be able to specify arbitrary pytest args when invoking tox